Makefile

```diff
@@ -15,13 +15,12 @@ clean: ## Remove build artifacts and .pyc files
 	-find . -type f -name '*.py[co]' -delete -o -type d -name __pycache__ -delete
 .PHONY: clean
 
-sdist: clean ## Build an sdist file
-	python setup.py sdist
+build: clean ## Build an sdist and wheel
+	python setup.py sdist bdist_wheel
 .PHONY: sdist
 
-upload: clean ## Upload an sdist file
-	python setup.py sdist upload
```

Comment:
If I am not mistaken, `setup.py sdist upload` means both "build an sdist and upload it" (rather than "upload an existing sdist"). This means "upload" also built the distribution. Do we want to keep this behavior and make the "upload" command run the "build" command? Or keep it like this so we need to run both "make build && make upload"? I'm fine either way.